.prettierrc

```diff
@@ -3,5 +3,6 @@
   "singleQuote": true,
   "trailingComma": "es5",
   "arrowParens": "always",
-  "parser": "babel"
+  "parser": "babel",
+  "endOfLine": "auto"
```

Comment:
pozera, ze toto sa nam este malicko lisi od kbc-ui, ale asi ok, ked tak doladime potom.

Reply:
Tusím ale to `"endOfLine": "auto"` tam zrovna máme no. Ale fajn to asi pak sjednotit.

Reply:
ok, to len ako upozornenie. Potom mozeme poriesit naraz.